app/src/main/java/com/infomaniak/drive/data/api/ApiRepository.kt

```diff
@@ -164,6 +167,39 @@ object ApiRepository : ApiRepositoryCore() {
         return callApi(ApiRoutes.getSession(driveId, uploadToken), DELETE, okHttpClient = okHttpClient)
     }
 
+    fun UploadFile.uploadUrl(uploadHost: String, chunkNumber: Int, currentChunkSize: Int): String {
```

Comment:
Passes the `driveId` and `uploadToken` as parameters instead of passing through a `UploadFile` extension.

Reply:
The `urls` are in `ApiRoutes`.